modules/http/src/test/java/org/eclipse/jetty/proxy/NTLMConnectHandler.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.eclipse.jetty.proxy;
```

Comment:
Are you replacing an existing implementation? If not just add this to our tests packages.